content/design-patterns/hq-style-games.textile

```diff
@@ -0,0 +1,65 @@
+---
+title: HQ style games
+section: design-patterns
+index: 31
+languages:
+  - none
+---
+
+HQ style apps have suddenly become popular. Here's how you can build your own HQ style applications.
+
+h2(#what). What are HQ style apps?
+
+In the recent past, a live quiz game called "HQ Trivia":https://en.wikipedia.org/wiki/HQ_Trivia has become quite popular. It's a TV game show, playable via a smartphone app that they offer. The idea is that a live TV show airs at a particular time while participants can play the live game for free using their smartphones. Participants have 10 seconds to try their luck, on each question asked.
+
+In such a scenario, speed and latency play huge roles. The app rewards winners with great amounts of money, hence, the tech behind the app cannot afford to go wrong. It gives quite a  unique experience to the users. This design pattern aims at explaining how one can build apps like this themselves, using Ably's realtime platform.
+
+h2(#how). Implementation details
+
+Ably deals with messages in real-time using named "channels":/realtime/channels-messages. For an HQ style app, we will require realtime messaging between the client and the server with minimal latencies. The questions could be sent from the server in the form of a video or just text data for the purpose of this design pattern. The timer is to be maintained by the server as well so that at any point during the game, the server remains to be the single source of truth, thus not letting client's network issues, if any, affect the realtime game flow.
```

Comment:
I've noticed the use of "in real-time" quite a lot recently in various docs / exmaples you have worked on. Please refer to the style guide, it's "in real time" and "realtime experiences"